tasks/copy_files_to_archive/README.md

```diff
@@ -179,7 +179,7 @@ DATA
     INPUT_FILENAME_KEY = 'filename'
     INPUT_GRANULE_ID_KEY = 'granule_id'
     INPUT_JOB_ID_KEY = 'job_id'
-    INPUT_MULTIPART_CHUNKSIZE_MB = 'multipart_chunksize_mb'
+    INPUT_MULTIPART_CHUNKSIZE_MB_KEY = 's3MultipartChunksizeMb'
```

Comment:
Need to revert value here of `'s3MultipartChunksizeMb'` to `'multipart_chunksize_mb'` to match what is in the code for copy_files_to_archive.py

Reply:
Some changes got lost, but at this point it's probably best not to require people to make changes to how they interact with lambdas. Reverted.